accountsplus/settings.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.apps import apps
+
+
+def get_setting(setting_str, default_value=None):
+    try:
+        return getattr(settings, setting_str, default_value)
```

Comment:
@nasief This will never raise an error because we are passing default_value parameter.